src/cr/cube/dimension.py

```diff
@@ -479,6 +487,14 @@ def shape(self):
         """int count of *all* elements in this dimension, both valid and missing."""
         return len(self.all_elements)
 
+    @lazyproperty
+    def subtotal_labels(self):
+        """tuple of string element-labels for each valid subtotal in this dimension.
+
+        Element-labels appear in the order defined in the cube-result.
+        """
+        return tuple(e.label for e in self.subtotals)
```

Comment:
This one I wasn't so sure about but I think I've come around.

`Subtotals` is a collection that is already used by callers, and I bet somewhere in the code we already have an expression like `s.label for s in dimension.subtotals` (might be worth looking for that and changing it)[Ah, I see you caught that one :)]. My initial thought would be to find these at `dimension.subtotals.labels`, but I think you're right:

1. `dimension.subtotal_labels` is parallel to `dimension.element_labels` and therefore less surprising than `dimension.subtotals.labels`.
2. The likelihood of needing any other `dimension.subtotal_{x}s` seems low, except maybe in future `.subtotal_element_ids`, but that would be justified by rationale 1 above (actually looks like we have `.insertion_ids` already, now that I look a second time).

Okay, cool, just took me a minute to talk myself into it.